### DISCUSSION TITLE
I want to read and answer from a csv file how can i do it?

### DISCUSSION THREAD

**Discussion by vortexai18** [FlowiseAI/Flowise#392] [Q&A]
I have loaded the csv file loader what text splitter should i use 
I have seen other places where to read from a csv file we need create_csv_agent which i am not able to find in agents. Please help me regarding this.

Thank you

**Comment by HenryHengZJ** [comment]
unfortunately this isnt supported on JS/TS yet as they `create-csv_agent` is using pandas dataframe under the hood which only support Python.

**Reply by HenryHengZJ** [reply]
having said that, there is a CSV loader in Flowise, try that:
![image](https://github.com/FlowiseAI/Flowise/assets/26460777/72d38ed1-e78e-4722-b354-10dbcb16a7d6)
